cmd/cosign/cli/options/root.go

```diff
@@ -32,6 +32,6 @@ func (o *RootOptions) AddFlags(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringVar(&o.OutputFile, "output-file", "",
 		"log output to a file")
 
-	cmd.PersistentFlags().BoolVarP(&o.Verbose, "verbose", "d", false,
-		"log debug output")
+	cmd.PersistentFlags().BoolVarP(&o.Verbose, "verbose", "v", false,
```

Comment:
mmk

Reply:
Which is more verbose?  verbose or debug? 🙃

Reply:
> Note this is an api breaking change on a v1.0 project. I would advise not making this change.

We haven't made any releases since `-d` was introduced :)

Reply:
Oh! It is easier then.

Reply:
@dekkagaijin should we create a milestone so we can assign what would be landing on 1.3 ?